script.js

```diff
@@ -1,4 +1,109 @@
+var win = 0;
+var loss = 0;
+var draw = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var num = numGenerator();
+  console.log(num);
+  var botOutput = botSpS(num);
+  console.log(botOutput);
+  var winLoseDraw = decisionMaker(input, botOutput);
+  console.log(winLoseDraw);
+
+  console.log(win);
+  console.log(loss);
+  console.log(draw);
+
+  return winLoseDraw;
+};
+
+var botSpS = function (num) {
+  if (num == 1) {
+    return "scissors";
+  }
+
+  if (num == 2) {
+    return "paper";
+  }
+
+  if (num == 3) {
+    return "stone";
+  }
+};
+
+var decisionMaker = function (input, botOutput) {
+  var input;
+  var botOutput;
+
+  if (!(input == "scissors" || input == "paper" || input == "stone")) {
+    return (
+      "Input invalid. Your input was " +
+      input +
+      " and the programme only accepts scissors, paper or stone (Caps sensitive). Please try again with valid input."
+    );
+  }
+  if (input == "scissors" && botOutput == "paper") {
+    win = win + 1;
+    return (
+      "You won! Your Win-Lose-Draw record is " + win + "-" + loss + "-" + draw
+    );
+  }
+  if (input == "paper" && botOutput == "stone") {
+    win = win + 1;
+    return (
+      "You won! Your Win-Lose-Draw record is " + win + "-" + loss + "-" + draw
+    );
+  }
+  if (input == "stone" && botOutput == "scissors") {
+    win = win + 1;
+    return (
+      "You won! Your Win-Lose-Draw record is " + win + "-" + loss + "-" + draw
+    );
+  }
```

Comment:
Nice use of conditionals. Can simplify the logic here by using the || operator
For example the win condition would look like this:

```
if ( input == "scissors" && botOutput == "paper" ||
input == "paper" && botOutput == "stone" ||
input == "stone" && botOutput == "scissors") {
// win
}
```